src/main/java/tutorease/address/logic/parser/ParserUtil.java

```diff
@@ -205,5 +206,11 @@ public static EndDateTime parseEndDateTime(StartDateTime startDateTime, String h
         return EndDateTime.createEndDateTime(startDateTime, hoursToAdd);
     }
 
-
+    /**
+     * Returns true if none of the prefixes contains empty {@code Optional} values in the given
+     * {@code ArgumentMultimap}.
```

Comment:
Do we need to add @params and @return for this javadocs

Reply:
This code and comment was extracted from the original AB3 as multiple commands uses it. Should I update the javadocs or keep it as it is?